app/lua/lobject.h

```diff
@@ -28,7 +28,7 @@
 #define LUA_TUPVAL	(LAST_TAG+2)
 #define LUA_TDEADKEY	(LAST_TAG+3)
 
-#ifdef LUA_USE_ESP
+#ifdef LUA_USE_ESP8266
```

Comment:
Does the ESP32 support unaligned data fetches from IRAM and IFLASH?.  If so then how many extra clocks?  The access macros add one extra register instruction to use an aligned read.  If this is still faster then why have this difference between the ESP8266 and ESP32 versions?

Reply:
Again, this is predominantly an Xtensa vs RISC-V issue here. But yes, data memory regions are [byte accessible](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/memory-types.html). Some optimisation may be possible, but has not been a priority for me at this point - getting NodeMCU to run on all the different ESP32 SoCs has been :)